src/main/java/codeu/controller/LoginServlet.java

```diff
@@ -84,6 +84,11 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     }
 
     request.getSession().setAttribute("user", username);
+
+    if (username.matches("admin") && password.matches("eastcode")) {
```

Comment:
So this code means that, those that are admin would have to use those credentials to be logged in as an admin?